include/helper/progress_helper.h

```diff
@@ -0,0 +1,28 @@
+#ifndef PROGRESS_HELPER_H
```

Comment:
Why is progress_helper.h in include/helper/ and not in itemframework/include/helper?